oxidation/crates/parsec_client_types/src/local_manifest.rs

```diff
@@ -184,6 +349,64 @@ impl From<LocalFolderManifest> for LocalFolderManifestData {
 
 impl_local_manifest_dump_load!(LocalFolderManifest);
 
+impl LocalFolderManifest {
+    pub fn _filter_local_confinement_points(&self) -> Self {
```

Comment:
protip: in Python methods starting with an underscore are private by convention, so no need to declare them `pub` in Rust ;-)

(an we can remove the underscore from the naming to look more Rust I guess ?)